composer.json

```diff
@@ -9,6 +9,12 @@
             "email": "haertl.mike@gmail.com"
         }
     ],
+    "require": {
+	"php": "^5.4.0 || ^7.0"
```

Comment:
How about a single `>=5.4.0` here?

Reply:
That's what I did initially, CI failed for php7. You can see why in the output of job 99.

Reply:
Hmm, i see a `^5.4.0`, not a `>=5.4.0` in your initial commit.